src/common/components/repository-row/index.js

```diff
@@ -44,6 +45,32 @@ class RepositoryRow extends Component {
     };
   }
 
+  saveState() {
+    localforage.setItem(`repository_row_${this.props.snap.git_repository_url}`, this.state);
+  }
```

Comment:
Unrelated question:

When do we start paying off technical debt? This task is not a "must have", but I hope we all agree we all want to start using the store as the state of the application where possible. I think it's worth the effort of adding a reducer and action creators for persisting row state if it's not due today.

(cc: @bartaz @earnubs @cjwatson)